src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard
```

Comment:
For consistency, I would keep this class in the same file unless there is another deriving class.

Reply:
Sounds good